app/operations/candidates/add.rb

```diff
@@ -129,6 +112,26 @@ def prepare_params(params:, actor_account:)
     params
   end
 
+  def add_events(candidate:, actor_account:)
+    Event.create!(
+      type: :candidate_added,
+      eventable: candidate,
+      performed_at: Time.zone.now,
```

Comment:
database have default value `performed_at timestamp without time zone DEFAULT clock_timestamp() NOT NULL`
may we omit setting current time?

Reply:
Looks like this way event's `update_candidate_last_activity` callback will not update `last_activity_at`

Reply:
No, actually, maybe our tests are just testing an impossible case. When we do time travel, postgres doesn't know that and inserts his own clock's now() that is literally now, unlike test's that is time where we traveled. But we can't really do time travel IRL and when we want to put custom performed_at we can just pass the necessary value. So maybe we could indeed remove passing `performed_at` and update the test. What do you think? Can we miss something this way?

Reply:
i like the idea of updating tests: make the candidate's last_activity_at in the past and after action check that it became today

but also we may add a callback into the event model `before_save` to set `performed_at ||= Time.zone.now` to allow us to use `travel_to` without problems (in case when we want to check event's performed_at or smth like this)

let's discuss on a team call